docs/API.md

```diff
@@ -1121,3 +1125,33 @@ A endpoint that can be used without the need for [authentication](#auth) which w
 HTTP GET /health
 
 It will output the state for each nodes in JSON, whose format might change in the future.
+
+## <a name="liquid"></a>Liquid integration
+
+NBXplorer supports liquid, the API is the same as all the other coins, except for the following:
+
+* All references to `value` which normally contains an integer of the amount of the altcoin will instead output a JSON Object of type `AssetMoney`.
+* [When listing the transaction of a derivation scheme](#transactions), the `balanceChange` elements is instead a `JSON array of AssetMoney`.
+* [Get Balance](#balance) returns values as `JSON array of AssetMoney`.
+* [Get a new unused address](#unused) returns a confidential address. (See note below)
+* [Create Partially Signed Bitcoin Transaction](#psbt) is not supported.
+* [Update Partially Signed Bitcoin Transaction](#updatepsbt) is not supported
+* [Scan UTXO Set](#scanUtxoSet) is not supported.
+* Any sort of recovery is not supported.
+
+The `AssetMoney` JSON format is:
+
+```json
+{
+	"assetId": "abc",
+	"value": 123
```

Comment:
How do I get the asset and amount blinding factors from this?

How about the case where I have a custom blinding key (e.g. I make my own addresses and do not trust NBXplorer's blinding key derivation scheme), i.e. where NBXplorer cannot unblind the output?

Reply:
if a tx is made to an address tracked by NBX, that is blinded by a key not available to NBX, it will be stored but its `Value`  property will be null.
I had a branch before where it would ignore any incoming txs where it could not unblind the tx.

Reply:
@kallewoof if you have your own derivation stuff, NBXplorer can't decrypt but you will get the outputs anyway with the blinded transaction, just not the Value.

Reply:
Yes, I realized after writing that you actually can't give me the blinding factors unless you know the blinding privkey. For the cases where you do, though, it would be helpful to include the blinding factors, as I could just pass those on when building a tx interactively with other people.